### ISSUE TITLE
This committee is shown in the Defense committee even though it is from the previous Knesset

### ISSUE THREAD

**Issue by morchickit** [hasadna/Open-Knesset#631]
https://oknesset.org/committee/meeting/9781/

Do we always show protocols of other Knessets in the current committee page?

**Comment by OriHoch** [comment]
yes, we show protocols from all knessets, you can see if you click on "more meetings"

for example: https://oknesset.org/committee/6/all_meetings/?&page=54

**Comment by alonisser** [comment]
@morchickit לא הבנתי מה הבעיה פה

**Comment by morchickit** [comment]
@alonisser - זה מבלבל, כי הפורטוקול היחיד לועדה הזו הוא מהכנסת הקודמת.

**Comment by alonisser** [comment]
צריך כאן החלטה, אולי אנחנו רוצים תת כותרת נפרדת לפני פרוטוקולים מכנסות
קודמות? אם זה מגיע ביחד בלי הפרדה אז הבעיה הזו יכולה לקרות.



Twitter:@alonisser <https://twitter.com/alonisser>
LinkedIn Profile <http://www.linkedin.com/in/alonisser>
Facebook <https://www.facebook.com/alonisser>
*Tech blog:*4p-tech.co.il/blog
*Personal Blog:*degeladom.wordpress.com
Tel:972-54-6734469

**Comment by morchickit** [comment]
כותרת נוספת יכול לעזור בהחלט.

**Comment by alonisser** [comment]
אוקי אז מציע לפתוח אישיו יעודי לזה שמגדיר את המקרה הכללי.  כולל טקסט והפניה
לכאן כדוגמא.  זה אולי יאפשר למישהו שלא מכיר את כל הדקויות פשוט לתקן את זה
בתאריך 22 במאי 2016 11:36 אחה״צ,‏ "morchickit" <notifications@github.com>
כתב:

> כותרת נוספת יכול לעזור בהחלט.
